app/parse-siti-payment-file/build-payment-file.js

```diff
@@ -0,0 +1,61 @@
+const readline = require('readline')
+const { Readable } = require('stream')
+
+const transformBatch = require('./transform-batch')
+const transformHeaders = require('./transform-headers')
+const transformInvoiceLines = require('./transform-invoice-lines')
+const buildPaymentRequests = require('./payment-request')
+const validate = require('./validate')
+
+const parseBatchLineType = (batchLine, batch) => {
+  const lineType = batchLine[0]
```

Comment:
There will be run time error if batchLine has 0 length. There should be a way of confirming that batchLine has length >0

Reply:
Aaahhhh I see what you mean but I ran a small experiment and JS returns `undefined` for `[]`, and `''` for `['']` which would be caught in the `default` case in our case statement. `'string'split('anything')` always returns an array with at least the original string, in this case `'string'` even if it's empty so `batchLine` will always be an array which can be 0-indexed.

If this code was to be reused (or potentially reused and exposed as a `module.export`) I would totally want this to be covered explicitly but I cnt test this scenario becuz I cnt this function as its not exposed (a tech debt ticket I have raised)